docs/devices/BSD-2.md

```diff
@@ -24,6 +24,11 @@ pageClass: device-page
 
 <!-- Notes BEGIN: You can edit here. Add "## Notes" headline if not already present. -->
 
+## Notes
+### Pairing
+- Before starting the pairing process the device's install code needs to be added to Zigbee2MQTT. Note down the install code e.g. by scanning the QR code with the camera of your mobile phone and copy the received value
+- The install code can be found inside the device before attaching it to the mounting plate
+- Add this install code under Settings -> Tools -> Add install code
```

Comment:
@N3rdix My BSD-2 has an install code of 9 blocks each with 4 hex characters printed on the back. That's 36 characters (excluding the `-` characters). 

If I paste that install code then I get a `Unsupported install code, got 36 chars, expected 95 or 91` error. My dongle is the `SONOFF Zigbee 3.0 USB Dongle Plus,TI CC2652P + CP2102(N)` and I am running zigbee2mqtt [1.36.1](https://github.com/Koenkk/zigbee2mqtt/releases/tag/1.36.1) commit: [ffc2ff1](https://github.com/Koenkk/zigbee2mqtt/commit/ffc2ff1).

Any idea why the pairing instructions don't work for me? (I've also asked about it [here](https://github.com/Koenkk/zigbee2mqtt/discussions/15966#discussioncomment-9019980))

Reply:
good catch, indeed the 9 blocks code is not the complete code you need (there are some other parts in front (like the ieee code between the QR code and the install code).
Easiest way I do this is scanning the QR code with my iPhone and copy that text

Reply:
@N3rdix I opened a PR to improve the instructions: https://github.com/Koenkk/zigbee2mqtt.io/pull/2684